catbuffer/schemas/nem/all_generated.cats

```diff
@@ -1,5 +1,2 @@
-# block
-import "block.cats"
-
```

Comment:
why did you remove this?

Reply:
Cause we've agreed, we _won't_ be dealing with NEM blocks?